camp/camp-brooklyn/src/main/java/org/apache/brooklyn/camp/brooklyn/spi/creation/BrooklynEntityDecorationResolver.java

```diff
@@ -46,22 +46,26 @@
 
 /**
  * Pattern for resolving "decorations" on service specs / entity specs, such as policies, enrichers, etc.
+ *
  * @since 0.7.0
  */
-@Beta
 public abstract class BrooklynEntityDecorationResolver<DT> {
 
     public final BrooklynYamlTypeInstantiator.Factory instantiator;
-    
+
     protected BrooklynEntityDecorationResolver(BrooklynYamlTypeInstantiator.Factory instantiator) {
         this.instantiator = instantiator;
     }
-    
+
+    /** @deprected since 0.10.0 - use {@link #decorate(EntitySpec<?>, ConfigBag, Set<String>)} */
+    @Deprecated
+    public abstract void decorate(EntitySpec<?> entitySpec, ConfigBag attrs);
```

Comment:
make this `public final` not abstract with the implementation passing the empty `ImmutableSet` ?